### PR TITLE
Feature/logout

### PR DESCRIPTION
**Pull request by adamsonb12**
Logout Endpoint, and some auto spacing things.

### CODE REVIEW

**Review by tjchecketts — DISMISSED** [review]

File: controllers/auth_controller/logout.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = (req, res) => {
+    req.logout();
+    req.session.destroy(err => {
+        if (!err) {
```
Comment (kyrandita) [inline controllers/auth_controller/logout.js]: flatten logic here
if(err){
 throw
}
...

File: guru.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = () => {
 
     guru.use(express.json());
 
-    guru.use(session({ secret: 'cats' }));
+    guru.use(session({ secret: 'guest' }));
```
Comment (kyrandita) [inline guru.js]: As funny as this is.... please reference an environment variable instead

File: services/passport.js [hunk 3]
```diff
@@ -16,8 +16,8 @@ passport.deserializeUser(async function(id, done) {
 });
 
 passport.use(
-    new LocalStrategy({usernameField: 'email'}, async function(email, password, done) {
-        try{
+    new LocalStrategy({ usernameField: 'email' }, async function(email, password, done) {
```
Comment (kyrandita) [inline services/passport.js:19]: :+1:

**Review by kyrandita — CHANGES_REQUESTED** [review]

File: guru.js (hunk 2 above)
Comment (adamsonb12) [reply guru.js]: We currently commit our .env, there is a task to remove that, and start using our .env correctly so we can store these types of values in it securely.

Comment (adamsonb12) [reply guru.js]: I can reference a variable to better prepare for those changes, which I'll do, but it won't be any more secure until then.

**Review by kyrandita — APPROVED** [review]
:black_heart: :+1:

**Pull request merged**